src/main/java/io/cdap/plugin/salesforce/plugin/source/batch/SalesforceBaseSourceConfig.java

```diff
@@ -120,7 +120,7 @@ protected SalesforceBaseSourceConfig(String referenceName,
                                        @Nullable String operation) {
     super(referenceName);
     this.connection = new SalesforceConnectorConfig(consumerKey, consumerSecret, username, password, loginUrl,
-                                                    securityToken, connectTimeout, oAuthInfo);
+      securityToken, connectTimeout, oAuthInfo);
```

Comment:
incorrect indentation. Have a check for all the changes.

Reply:
done. Few more files got added to commit when checking for indentation.